openhands/runtime/impl/eventstream/eventstream_runtime.py

```diff
@@ -117,6 +117,7 @@ def __init__(
         self.config = config
         self._host_port = 30000  # initial dummy value
         self._container_port = 30001  # initial dummy value
+        self.default_port_mapping = {4141: 4141, 4142: 4142}
```

Comment:
I have a question: if I'd like to run OpenHands without any port mapping (e.g. no enough privilege, or due to security concerns), is that still possible with this default port mapping stuff?

Reply:
Not at the moment, this is a temporary solution. In the future, we will allow the user full control of what ports (if any) they want to forward